pimd/pim_upstream.c

```diff
@@ -830,8 +833,11 @@ void pim_upstream_fill_static_iif(struct pim_upstream *up,
 
 	/* reset other parameters to matched a connected incoming interface */
 	up->rpf.source_nexthop.mrib_nexthop_addr.family = AF_INET;
```

Comment:
Wrong AF here, need `PIM_AF`

Reply:
Actually please replace 6 lines with 1:
```
pim_addr_to_prefix(&up->rpf.source_nexthop.mrib_nexthop_addr, PIMADDR_ANY);
```
That way we don't need the `#if` and can't have accidental wrong AF like happened here :smile:

(repeated below)

Reply:
Thanks for catching this :)